### PR TITLE
feature: using generators instead of range() in avoid-those-slow-functions rule

### PR DESCRIPTION
**Pull request by aleksandrzen**

### CODE REVIEW

**Comment by aleksandrzen** [comment]
There is a typo in commit's comment, of course I suggested to use generators instead of `range()`, not `sort()` function.

**Comment by dseguy** [comment]
aleksandrzen : +1. Can you redo the PR? I can't merge it from the interface.

File: rules/avoid-those-slow-functions.md [hunk 1]
```diff
@@ -13,19 +13,16 @@ There are a few PHP native functions that should be avoided for speed reasons. T
...
 | strstr             | Use strpos() for simple searches |
 | uniqid()           | Always mention entropy (2nd parameter) |
-| array\_walk()       | Use foreach($source as &$variable) { } |
+| array\_walk()      | Use foreach($source as &$variable) { } |
 | array\_map()       | Use foreach($source as &$variable) { } |
-
-<!--
-|   |   |
--->
+| range()            | You can use [generators](http://php.net/manual/ru/language.generators.overview.php), for preventing building an array in memory|
```
Comment (ArnaudLigny) [inline rules/avoid-those-slow-functions.md]: Could you change to a "generic" URL (without "ru/")?

Comment (aleksandrzen) [inline rules/avoid-those-slow-functions.md]: @Narno, :+1:

**Pull request merged**